<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontWidget</class>
 <widget class="QWidget" name="FontWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Atlas Typeface:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="PropertiesTreeView" name="typefaceProperties"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpFontState">
       <property name="title">
        <string>Font:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QComboBox" name="cmbFontList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbSize">
            <property name="suffix">
             <string> pt</string>
            </property>
            <property name="minimum">
             <number>4</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Layers:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QToolButton" name="btnAddLayer">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbRenderMode">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="currentIndex">
                <number>-1</number>
               </property>
               <property name="maxVisibleItems">
                <number>5</number>
               </property>
               <property name="maxCount">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="sbThickness"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="FontTableView" name="layersTableView">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectItems</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QToolButton" name="btnRemoveLayer">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="btnOrderLayerUp">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnOrderLayerDown">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddLayer">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>AddLayer</string>
   </property>
   <property name="toolTip">
    <string>Add a new layer to this font state</string>
   </property>
  </action>
  <action name="actionRemoveLayer">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>RemoveLayer</string>
   </property>
   <property name="toolTip">
    <string>Remove the currently selected layer from the font state</string>
   </property>
  </action>
  <action name="actionOrderLayerDownwards">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-down.png</normaloff>:/icons16x16/generic-down.png</iconset>
   </property>
   <property name="text">
    <string>Order Layer &amp;Downwards</string>
   </property>
   <property name="toolTip">
    <string>Sort the selected layer back one</string>
   </property>
  </action>
  <action name="actionOrderLayerUpwards">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-up.png</normaloff>:/icons16x16/generic-up.png</iconset>
   </property>
   <property name="text">
    <string>Order Layer &amp;Upwards</string>
   </property>
   <property name="toolTip">
    <string>Sort the selected layer up one</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertiesTreeView</class>
   <extends>QTreeView</extends>
   <header>PropertiesTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>FontTableView</class>
   <extends>QTableView</extends>
   <header>FontModels.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/HyEditor.qrc"/>
 </resources>
 <connections/>
</ui>
